lib/orwell/web/router.ex

```diff
@@ -28,6 +28,8 @@ defmodule Orwell.Web.Router do
     delete "/auth", AuthController, :delete
     get "/auth/:provider", AuthController, :request
     get "/auth/:provider/callback", AuthController, :callback
+
+    get "/*path", PostController, :show
```

Comment:
It's way simpler to just use the path.

GitHub API requires the full path to retrieve the file contents. In the post index page you just need to pass the path returned by GitHub to the link generator to get a proper link. And then, on the post controller's `show` action you have the full path and you just need to pass it along to the `GitHub.Blog` module.

If we go with `posts/id` then you need to parse every URL in the index page to only retrieve the ID. In the `show` action, you need to collect every file from the git repository, filter down the posts, see which one matches the ID and do a second request just to retrieve the file contents. It's less maintainable and more costly since you are making 2 API requests.

I can have it dinamically match the `/pages/posts/path-to-post.md` instead.